kyuubi-common/src/test/scala/org/apache/kyuubi/HudiSuiteMixin.scala

```diff
@@ -42,5 +42,6 @@ trait HudiSuiteMixin extends DataLakeSuiteMixin {
     "spark.sql.defaultCatalog" -> catalog,
     "spark.sql.extensions" -> "org.apache.spark.sql.hudi.HoodieSparkSessionExtension",
     "spark.serializer" -> "org.apache.spark.serializer.KryoSerializer",
+    "hoodie.bootstrap.index.class" -> "org.apache.hudi.common.bootstrap.index.HFileBootstrapIndex",
```

Comment:
Actually, `HFileBootstrapIndex` is the default value, it's not matter with/without this conf I think.

Reply:
Yes, HFileBootstrapIndex is the default value, if you want reduce hbase dep, this config should be 'org.apache.hudi.common.bootstrap.index.NoOpBootstrapIndex'.

Reply:
Yeah, just change it back to default for test. BTW, is it the right approach to overwrite hudi conf?

Reply:
Overwriting the config when creating the table

https://hudi.apache.org/docs/quick-start-guide
![image](https://user-images.githubusercontent.com/18065113/132175272-03cd3784-7c04-467e-bd71-cb6a89a77ad4.png)

Reply:
Thanks, we need refactor `DataLakeSuiteMixin` and expose create table options to subclass in future.